iis/mymodule.cpp

```diff
@@ -798,7 +798,9 @@ CMyHttpModule::OnBeginRequest(IHttpContext* httpContext, IHttpEventProvider* pro
     HRESULT hr = MODSECURITY_STORED_CONTEXT::GetConfig(httpContext, &config);
     if (FAILED(hr))
     {
-        return RQ_NOTIFICATION_CONTINUE;
+        httpContext->GetResponse()->SetStatus(500, "WAF internal error");
```

Comment:
Should we say something like "IWAF configuration error" here to give a hint to what is wrong?

Reply:
the thing is, the user didn't do anything wrong here. We, internally in our code, generated an invalid modsecurity config.

Reply:
will use this compromise: WAF internal error. Invalid configuration.